src/Components/Shared/src/ElementReferenceJsonConverter.cs

```diff
@@ -30,7 +30,7 @@ public override ElementReference Read(ref Utf8JsonReader reader, Type typeToConv
                 }
                 else
                 {
-                    throw new JsonException($"Unexcepted JSON Token {reader.TokenType}.");
+                    throw new JsonException($"Unexpected JSON Token {reader.TokenType}.");
```

Comment:
@pranavkm this is an existing exception message change. Is it safe to tweak this?

Reply:
Yup, this should be fine